server_admin/topics/realms/keys.adoc

```diff
@@ -8,7 +8,7 @@ encryption.  {project_name} uses asymmetric key pairs, a private and public key,
 is used to create new signatures, while the passive keypair can be used to verify previous signatures. This makes it
 possible to regularly rotate the keys without any downtime or interruption to users.
 
-When a realm is created a key pair and a self-signed certificate is automatically generated.
+When a realm is created, a keypair and a self-signed certificate is automatically generated.
```

Comment:
If we are referring to the Java class, I'd suggest to use "KeyPair". Otherwise, I believe it is correct to use "key pair".